pkg/user/password_reset_test.go

```diff
@@ -0,0 +1,99 @@
+package user
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewPasswordReset(t *testing.T) {
+	pr := NewPasswordReset()
+	assert.NotNil(t, pr)
+	assert.NotEmpty(t, pr.Token)
+	assert.True(t, time.Until(pr.CreatedAt) < 1*time.Second)
```

Comment:
You can use https://pkg.go.dev/github.com/stretchr/testify/assert#Less

Reply:
Less can not compare type "time.Time"

Reply:
```suggestion
	pr := NewPasswordReset()
	assert.NotNil(t, pr)
	assert.NotEmpty(t, pr.Token)
	assert.Less(t, int64(time.Until(pr.CreatedAt)), int64(1*time.Second))
```